generators/php/base/src/asIs/Client/RawClient.Template.php

```diff
@@ -247,4 +261,21 @@ private function encodeQueryValue(mixed $value): string {
         // Unreachable, but included for a best effort.
         return urlencode(strval(json_encode($value)));
     }
+
+    /**
+     * Check if an array is sequential, not associative.
+     * @param array $arr
+     * @return bool
+     */
+    private static function isSequential(array $arr): bool { // @phpstan-ignore-line
```

Comment:
What are we ignoring here?

Reply:
phpstan wants `array` to have more information, but we don't have more info